src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,396 @@
+# Microscopy
+
+Support for Microscopy was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+<!---
+The following link is temporary and points to the bep031 branch of bids-examples,
+update to 'https://github.com/bids-standard/bids-examples' when examples are merged.
+-->
+Microscopy datasets formatted using this specification are available on the
+[BIDS examples repository](https://github.com/neuropoly/bids-examples/tree/bep031)
+(`microscopy_SEM001` and `microscopy_SPIM001`) and can be used for practical
+guidance when curating a new dataset.
+
+Further Microscopy datasets are available:
+
+-   In PNG format:  [`data_axondeepseg_sem`](https://doi.org/10.5281/zenodo.5498378)
+-   In OME-TIFF format: [`Broca's Area Light-Sheet Microscopy`](https://doi.org/10.5281/zenodo.5517223)
+
+## Microscopy recording data
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+Template:
+```Text
+sub-<label>/
+    [ses-<label>/]
+        microscopy/
+            sub-<label>[_ses-<label>]_sample-<label>[_acq-<label>][_stain-<label>][_run-<index>][_chunk-<index>]_<suffix>.<extension>
+            sub-<label>[_ses-<label>]_sample-<label>[_acq-<label>][_stain-<label>][_run-<index>][_chunk-<index>]_<suffix>.json
+```
+Microscopy data MUST be stored in the `microscopy` directory.
+
+### File formats
+The Microscopy community uses a variety of formats for storing raw data, and there is no single
+standard that all researchers agree on. However, a standardized file structure has been developed
+by the [Open Microscopy Environment](https://www.openmicroscopy.org/) for whole-slide imaging with
+the [OME-TIFF file specifications](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/file-structure.html).
+The OME-TIFF file allows for multi-page TIFF files to store multiple image planes and supports
+multi-resolution pyramidal tiled images. An OME-XML data block is also embedded inside the
+file’s header.
+
+In Microscopy-BIDS, we accept different file formats to accommodate datasets stored in 2D image
```

Comment:
The usage of `Microscopy-BIDS` is awkward, and inconsistent with this being integrated into the BIDS standard as a whole.   I think you want text more like:

> The BIDS standard accepts microscopy data in different file formats to accommodate ...